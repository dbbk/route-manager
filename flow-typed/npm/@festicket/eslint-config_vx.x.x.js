// flow-typed signature: 564086c0274f153b4c37ac5c954f219e
// flow-typed version: <<STUB>>/@festicket/eslint-config_v^1.1.5/flow_v0.63.1

/**
 * This is an autogenerated libdef stub for:
 *
 *   '@festicket/eslint-config'
 *
 * Fill this stub out by replacing all the `any` types.
 *
 * Once filled out, we encourage you to share your work with the
 * community by sending a pull request to:
 * https://github.com/flowtype/flow-typed
 */

declare module '@festicket/eslint-config' {
  declare module.exports: any;
}

/**
 * We include stubs for each file inside this npm package in case you need to
 * require those files directly. Feel free to delete any files that aren't
 * needed.
 */


// Filename aliases
declare module '@festicket/eslint-config/index' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
declare module '@festicket/eslint-config/index.js' {
  declare module.exports: $Exports<'@festicket/eslint-config'>;
}
