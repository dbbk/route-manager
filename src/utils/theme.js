// @flow
import { darken } from 'polished';

export function hoverColor(color: string): string {
  return darken(0.05, color);
}

export default {
  colors: {
    white: 'white',
    success: '#57C0B4',
    danger: '#DE5E5E',
    dangerBackground: '#FDF6F6', // primary with transrency 5% on white background
    info: '#A3A6CB',
    star: '#FFCD00',
    background: {
      secondary: '#F9F9FC',
    },
    greyscale: {
      black: '#242529',
      dark: '#A2A4AD',
      silver: '#E2E3E6',
      grey: '#F5F5F5',
    },
    brand: {
      primary: '#8C64CB',
      primaryBackground: '#F9F7FC', // primary with transrency 5% on white background
    },
    facebook: {
      primary: '#4B69AC',
    },
    shadowColor: 'rgba(36, 37, 41, 0.15)',
  },
  animationDuration: {
    shortest: '50ms',
    short: '350ms',
  },
  fontFamily: 'brandon-text, Helvetica Neue, Helvetica, Arial, sans-serif',
  fontWeight: {
    light: '300',
    regular: '400',
    bold: '700',
  },
  breakpoints: {
    xs: 0,
    sm: 568,
    md: 768,
    lg: 960,
    xlg: 1220,
  },
  clipPath: {
    polygonBottomLarge: 'polygon(0 0, 100% 0, 100% 100%, 0 85%)',
  },
};
