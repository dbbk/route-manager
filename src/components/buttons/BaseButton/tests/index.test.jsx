// @flow

import React from 'react';
import { shallow } from 'enzyme';
import { BaseButton } from '@festicket/react-ui-components';
import { ThemeProvider } from 'styled-components';
import theme from '../../../../utils/theme';

describe('<BaseButton />', () => {
  test('with initial props', () => {
    const tree = shallow(
      <ThemeProvider theme={theme}>
        <BaseButton />
      </ThemeProvider>,
    )
      .first()
      .shallow();
    expect(tree).toMatchSnapshot();
  });

  test('with href', () => {
    const tree = shallow(
      <ThemeProvider theme={theme}>
        <BaseButton href="https://www.google.com" />
      </ThemeProvider>,
    )
      .first()
      .shallow();
    expect(tree).toMatchSnapshot();
  });
});
