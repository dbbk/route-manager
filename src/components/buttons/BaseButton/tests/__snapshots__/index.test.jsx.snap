// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BaseButton /> with href 1`] = `
<Styled(BaseButtonComponent)
  href="https://www.google.com"
  size="responsive"
  type="button"
  variant="primary"
/>
`;

exports[`<BaseButton /> with initial props 1`] = `
<Styled(BaseButtonComponent)
  size="responsive"
  type="button"
  variant="primary"
/>
`;
